frontend/lib/tests/mock-fetch.ts

```diff
@@ -1,10 +1,8 @@
-type FetchAPI = typeof window.fetch;
-
 /**
  * This is an API for a mock window.fetch, which provides
  * some convenience methods for network communications.
  */
-interface MockFetch extends jest.MockInstance<FetchAPI> {
+interface MockFetch extends jest.MockInstance<any, any> {
```

Comment:
I don't really understand what's going on with the typing of `jest.MockInstance` but it's just for tests so I don't care too much. 🤷‍♂️